UHResidentInformationAPI/V1/Infrastructure/ContactLink.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("cccontactlink")]
+    public class ContactLink
+    {
+        [Column("contactno")]
+        [Key]
+        public int ContactID { get; set; }
+
+        [Column("key1")]
+        [StringLength(20)]
+        public string TagRef { get; set; }
```

Comment:
As per above - remains to be seen if TagRef is the right thing to call this!
Discussions shall happen :)